lib/sensors/MoistureSensor.h

```diff
@@ -0,0 +1,20 @@
+#include "mbed.h"
+#include <I2C.h>
+
+#define I2C_Addr 0x36
```

Comment:
choose a more descriptive name. also prefer `constexpr` instead of macros for type safety